src/Shared/UnitTests/App.config

```diff
@@ -30,6 +30,15 @@
         <assemblyIdentity name="Microsoft.Build.Utilities.Core" culture="neutral" publicKeyToken="b03f5f7f11d50a3a" />
         <bindingRedirect oldVersion="0.0.0.0-99.9.9.9" newVersion="15.1.0.0"/>
       </dependentAssembly>
+
+      <!-- RuntimeInformation is redirected because src/Build.UnitTests/project.json incorrectly resolves to 4.0.0.0 (package 4.0.0-rc3-24117-00) 
+      instead of 4.0.1.0 (package 4.3.0) for net4.6.
```

Comment:
It is really weird. The Utilities unit test have an almost identical project.json and an identical dependency tree path for RuntimeInformation, but they get 4.30 whereas the engine unit test project.json gets 4.0.0. I looked over it with @kingerja but could not figure it out.

Maybe upgrading our CLI version to the latest one that still supported project.jsons would fix it, but probably it's worth just waiting for the SDKs :)